### DISCUSSION TITLE
Não está fazendo corretamente o cast Req.Body<TJsonObject>

### DISCUSSION THREAD

**Discussion by ademarrs** [HashLoad/horse#295] [Q&A] [answered]
Estou implementando os exemplos do curso do CRUD de cadastro de produtos, o endpoint de cadastro do produto não está funcionando, por que o cast do Body para um JsonObjet, não está funcionando, sempre retorna NIL.
![Horse Cad produto](https://user-images.githubusercontent.com/29805902/185123610-319afc95-3d4f-4eb0-a6f3-1719c242709b.png)

**Comment by alcantarajoaopaulo** [comment]
Olá,
Favor, informar o endpoint que está usando a procedure CadastrarProduto (por exemplo App.Get, App.Post, etc) e postar também o conteúdo que está sendo enviado do seu aplicativo que faz a requisição (Postman, Insomnia, RESTDebugger, ou client próprio)

**Reply by ademarrs** [reply]
Estou usando o método Post
![Horse Post](https://user-images.githubusercontent.com/29805902/185140990-09f9d5d1-3bb0-4dfe-b102-39e091fe1546.png)

**Comment by alcantarajoaopaulo** [comment, marked as answer]
Os testes aqui funcionaram normalmente. Única diferença foi no Postman escolher a opção raw com o tipo JSON em vez de Text.

Confirmando as versões do Horse e do Jhonson no seu boss.json:

"github.com/hashload/horse": "^3.0.2",
"github.com/hashload/jhonson": "^1.1.4"

Está correto?

**Reply by ademarrs** [reply]
Sim, estão nessa versão.
Fiz o teste mudando de Text para Json, no PostMan, e deu certo.

Obrigado
